torchvision/models/detection/anchor_utils.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+import torch
+import torchvision
+from torch import nn
+
+from torch.jit.annotations import List, Optional, Dict
+
+
+class AnchorGenerator(nn.Module):
+    __annotations__ = {
+        "cell_anchors": Optional[List[torch.Tensor]],
+        "_cache": Dict[str, List[torch.Tensor]]
+    }
+
+    """
+    Module that generates anchors for a set of feature maps and
+    image sizes.
+    The module support computing anchors at multiple sizes and aspect ratios
```

Comment:
nit: There are a few newlines missing compared to the previous version, which would make the documentation not render properly. Can you fix it?

Reply:
Good catch! Strange enough when copying from the github interface it skips those newlines ... I copied the class properly now.

Also, please continue be nitty, it is very important for the health of the code in my opinion :+1: